src/valkey-benchmark.c

```diff
@@ -1557,6 +1509,12 @@ int parseOptions(int argc, char **argv) {
         "                    If the command is supplied on the command line in cluster\n"
         "                    mode, the key must contain \"{tag}\". Otherwise, the\n"
         "                    command will not be sent to the right cluster node.\n"
+        " --rfr <mode>       Enable read from replicas in cluster mode.\n"
+        "                    This command must be used with the --cluster option.\n"
+        "                    When using this option, it is recommended to use only \n"
+        "                    the commands for read requests.\n"
+        "                    default=read from replica only\n"
+        "                    2=read from all nodes(primary and replica)\n"
```

Comment:
Lets fix this help section:
1. I think the command should accept string values eg 'primaries' (default), 'replicas', 'all' 
2. Lets make sure we socument all possible values here 
3. IIUC the default is read from primaries only right?

Reply:
Good comments.
1. I'll fix it to string value such as "replicas" and "all". It's weird that “primaries” is the default, as you mentioned below, and I'd rather use `goto invalid`.
2. I agree.
3. If the user run it with `--cluster` only, the default is reading from primaries only. If the user run with `--cluster --rfr`, the default is reading from replicas only.